Makefile

```diff
@@ -35,30 +70,30 @@ release: \
 # matchbox
 
 bin/linux-amd64/matchbox:
```

Comment:
I forgot that you could only have one per target. You'd need to write this:

```make
bin/linux-amd64/matchbox: OS=linux
bin/linux-amd64/matchbox: ARCH=amd64
bin/linux-arm/matchbox: OS=linux
bin/linux-arm/matchbox: ARCH=arm
...

bin/%/matchbox:
        GOOS=$(OS) GOARCH=$(ARCH) go build -o $@ -ldflags $(LD_FLAGS) -a $(REPO)/cmd/matchbox
```

Or you could combine all of these into a single variable:

```make
bin/linux-amd64/matchbox: GOARGS = GOOS=linux GOARCH=amd64
bin/linux-arm/matchbox: GOARGS = GOOS=linux GOARCH=arm
...

bin/%/matchbox:
        $(GOARGS) go build -o $@ -ldflags $(LD_FLAGS) -a $(REPO)/cmd/matchbox
```